static/azure.hbs

```diff
@@ -21,4 +21,10 @@ steps:
   # # Start local server
   # - script: npm run start & npx wait-on http://localhost:3000
 
-  - script: npm test
\ No newline at end of file
+  {{#if qawolf}}
+  - script: npx qawolf test
+  {{else}}
+  # replace below with command you want to run, example for running a script below
```

Comment:
I would change this to: "example for running a script/test below"

and have the npm test commented out too

Reply:
I'm in the camp of keeping the default obvious, they can change if they want

Reply:
Ok 👍